modules/aws_k8s_base/tf_module/ingress_nginx.tf

```diff
@@ -43,7 +43,7 @@ resource "helm_release" "ingress-nginx" {
   chart            = "ingress-nginx"
   name             = "ingress-nginx"
   repository       = "https://kubernetes.github.io/ingress-nginx"
-  version          = contains(["1.19", "1.20", "1.21", "1.22"], var.k8s_version) ? "4.1.4" : "3.40.0"
```

Comment:
Not sure that this was necessary given that that validator from `aws_eks` ensures that `k8s_version` is one of these values already. So this conditional is always true.